packages/design-system/theme/src/button.scss

```diff
@@ -28,17 +27,29 @@ $loading-overlay-background-color: rgba(255, 255, 255, 0.35);
 	box-sizing: border-box;
 	outline: none;
 	margin: 0;
-	transition: 0.1s;
+	transition: 0.3s;
 
 	@include utils.utils-user-select(none);
 
+
+  	&:hover {
+		color: var.$button-hover-color;
+		border-color: var.$button-hover-border-color;
+		background-color: var.$button-hover-background-color;
+  	}
+
 	&:active {
 		color: var.$button-active-color;
 		border-color: var.$button-active-border-color;
 		background-color: var.$button-active-background-color;
 		outline: none;
 	}
 
+  	&:focus {
+	  	border-color: var.$button-focus-outline-color;
+	    outline: 2px solid var.$button-focus-outline-color;
```

Comment:
Would it be beneficial to extract this outline width as a variable to we can use it for other components focus style?

Reply:
Great suggestion. Done! :)